rsc/src/main/scala/rsc/scalasig/Pickle.scala

```diff
@@ -1341,4 +1340,17 @@ object Pickle {
   def apply(settings: Settings, mtab: Mtab, sroot1: String, sroot2: String): Pickle = {
     new Pickle(settings, mtab, sroot1, sroot2)
   }
+
+  final implicit class ValueOps(val value: String) extends AnyVal {
+    def encode: String = {
+      if (value == "_root_") "<root>"
+      else if (value == "_empty_") "<empty>"
+      else if (value == "<init>") "<init>"
```

Comment:
This is super minor but if you changed it to 
```
else if (value == "<init>") value 
```
Then it would make it more obvious to the reader that no transformation is being done.